docs/operate/versioning.mdx

```diff
@@ -11,147 +11,172 @@ import Admonition from '@theme/Admonition';
+To check which deployments are currently serving your services, do the following:
 
 <Tabs groupId="interface" queryString>
-<TabItem value="cli" label="CLI">
+    <TabItem value="cli" label="CLI">
 
-```shell
-restate dp add http://greeter-v2/
-```
+        ```bash
+        restate services list
+        ```
 
-</TabItem>
-<TabItem value="curl" label="curl">
 
-```bash
-curl <RESTATE_ADMIN_URL>/deployments -H 'content-type: application/json' -d '{"uri": "http://greeter-v2/"}'
-```
+        <details className={"grey-details"}>
+            <summary>Output</summary>
 
-This returns:
-
-```json
-{
-    "id": "Z3JlZXRlci12Mi8",
-    "services": [
-{
-    "name": "greeter.Greeter",
-    "revision": 2
-}
-    ]
-}
-```
+            ```shell
+            NAME             REVISION  FLAVOR  DEPLOYMENT TYPE  DEPLOYMENT ID
+            # mark[57:82]
+            🌎  TicketObject     1                 HTTP 2           dp_14LsPzGz9HBxXIeBoH5wYUh
+            # mark[57:82]
+            🌎  CartObject       1                 HTTP 2           dp_14LsPzGz9HBxXIeBoH5wYUh
+            # mark[57:82]
+            🌎  CheckoutService  1                 HTTP 2           dp_14LsPzGz9HBxXIeBoH5wYUh
+            ```
 
-</TabItem>
-</Tabs>
+        </details>
 
-This notifies that Restate detected a new revision of an already existing service, and from now on it will route any invocation of `greeter.Greeter` to `http://greeter-v2/`, while any existing invocation to `greeter.Greeter`, e.g. an invocation sleeping for some time, will continue executing on `http://greeter-v1/` until the end.
+        To list only the deployments, you can do `restate deployments list`.
 
-To check which endpoint is currently serving new invocations of `greeter.Greeter`:
+        Note that deployment IDs start with `dp_`. You can use it to describe the deployment:
 
-```bash
-curl <RESTATE_ADMIN_URL>/services/greeter.Greeter
-```
+        ```bash
+        restate deployment describe dp_14LsPzGz9HBxXIeBoH5wYUh
+        ```
 
-This returns:
+    </TabItem>
 
-```json
-{
-    "name": "greeter.Greeter",
-    "revision": 2,
-    "endpoint_id": "Z3JlZXRlci12Mi8",
-    [...]
-}
-```
+    <TabItem value="curl" label="curl">
 
-To get more info about the deployment serving it:
+        ```bash
+        curl localhost:9070/services/Greeter
+        ```
 
-```bash
-curl <RESTATE_ADMIN_URL>/deployments/Z3JlZXRlci12Mi8
-```
+        <details className={"grey-details"}>
+            <summary>Output</summary>
 
-```json
-{
-    "id": "Z3JlZXRlci12Mi8",
-    "uri": "http://greeter-v2/",
-    "additional_headers": {},
-    "protocol_type": "BidiStream",
-    "services": [
-      {
-        "name": "greeter.Greeter",
-        "revision": 2
-      }
-    ]
-}
-```
+            ```json
+            {
+                "name": "Greeter",
+                "revision": 2,
+                // withClass highlight-line
+                "endpoint_id": "Z3JlZXRlci12Mi8",
+                [...]
+            }
```

Comment:
This doesn't look correct anymore. I think it should be `deployment_id` and it should be a deployment id of the form `dp_...`.